.github/workflows/maven.yml

```diff
@@ -16,16 +16,18 @@ on:
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
-
     steps:
-    - uses: actions/checkout@v3
-    - name: Set up JDK 11
-      uses: actions/setup-java@v3
-      with:
-        java-version: '11'
-        distribution: 'temurin'
-        cache: maven
-    - name: Build with Maven
-      run: mvn -B package --file pom.xml
+      - uses: actions/checkout@v3
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'temurin'
+          cache: maven
+      - name: Build with Maven
+        run: mvn -B package --file pom.xml
+      - name: Build and test
+        run: mvn verify
+      - name: Run Checkstyle
```

Comment:
You shouldn't need to run the `checkstyle` checks separately as they will run as part of `mvn -B package --file pom.xml`

Reply:
These lines have been removed